src/md/src/Analysis/AutoAnalysisOfLaunches.md

```diff
@@ -42,90 +40,249 @@ The following info is sent:
 * A launch name;
 * Unique ID;
 
-So that the more you analyze, the more data in the ElasticSearch, the more accurate results you will get in the end of the process. 
+Service Analyzer forms a query with following data to the  service ElasticSearch.:
+* LogMessage;
+* Launch ID;
+* UniqueID test item; 
+* Defect type;
+
+So each log with its defect type is saved in ElasticSearch as a separate document. All document creates Index. So the more test results you analyze , the more data in the ElasticSearch appears, the more accurate results you will get in the end of the process. 
```

Comment:
Each log entry along with it's defect type is saved to ElasticSearch in form of separate document.

Reply:
The documents created compose an index. The more test results index has, the more accurate results will be generated by the end of the process.